meta-lmp-base/recipes-bsp/u-boot/u-boot-fio-mfgtool_2019.10.bb

```diff
@@ -2,7 +2,7 @@ SUMMARY = "Produces a Manufacturing Tool compatible U-Boot"
 DESCRIPTION = "U-Boot recipe that produces a Manufacturing Tool compatible \
 binary to be used in updater environment"
 
-require recipes-bsp/u-boot/u-boot-fio_git.bb
+require recipes-bsp/u-boot/u-boot-fio_2019.10.bb
```

Comment:
In some other patch, it was used ${PV} instead of `_2019.10`

Reply:
I don't see a reason to use PV here. IPV will always be "2019.10" in this recipe.
Using PV makes sense in wildcarded recipes IMHO.

Reply:
ok @MrCry0